packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss

```diff
@@ -1,7 +1,10 @@
 .tabs-container {
   margin: 0;
   width: 100%;
-  box-shadow: 0 -1px 0 rgba($ui-02, 0.25);
+  background: $ui-05;
```

Comment:
Same as the above question, should be black not gray-100?

Reply:
same, switched it back